packages/amplify-frontend-flutter/lib/configuration-manager.js

```diff
@@ -0,0 +1,71 @@
+const inquirer = require('inquirer');
+const constants = require('./constants');
+
+async function init(context) {
+  normalizeInputParams(context);
+  await confirmConfiguration(context);
+  return context;
+}
+
+function onInitSuccessful(context) {
+  return context;
+}
+
+async function configure(context) {
+  normalizeInputParams(context);
+  await confirmConfiguration(context);
+  return context;
+}
+
+function normalizeInputParams(context) {
+  let inputParams;
+  if (context.exeInfo.inputParams && context.exeInfo.inputParams[constants.Label]) {
+    inputParams = context.exeInfo.inputParams[constants.Label];
+  }
+  if (inputParams && inputParams.config) {
+    if (!inputParams.config.ResDir) {
+      throw new Error('The command line parameter for Flutter frontend configuration is incomplete.');
+    }
+  }
+  context.exeInfo.inputParams[constants.Label] = inputParams;
+}
+
+async function confirmConfiguration(context) {
+  if (!context.exeInfo.projectConfig[constants.Label]) {
+    context.exeInfo.projectConfig[constants.Label] = {};
+  }
+  if (!context.exeInfo.projectConfig[constants.Label].config) {
+    context.exeInfo.projectConfig[constants.Label].config = {};
+  }
+
+  const inputParams = context.exeInfo.inputParams[constants.Label];
+  if (inputParams) {
+    Object.assign(context.exeInfo.projectConfig[constants.Label], inputParams);
+  } else if (!context.exeInfo.inputParams.yes) {
+    context.print.info('Please tell us about your project');
+    const { config } = context.exeInfo.projectConfig[constants.Label];
+
+    context.print.warning('⚠️  Flutter project support in the Amplify CLI is in DEVELOPER PREVIEW.');
+    context.print.warning('Only the following categories are supported:');
+    context.print.warning(' * Auth');
+    context.print.warning(' * Analytics');
+    context.print.warning(' * Storage');
```

Comment:
In Analytics, we have Kinesis there, which is not supported, only Pinpoint, right? Perhaps we should pointing it out.

Also why do we have API, Predictions related code (`constructPredictions`) if that is not supported? Please remove the excess.

Reply:
Done.